files/fr/web/media/formats/image_types/index.md

```diff
@@ -891,7 +891,7 @@ Le JPEG est en fait un format de données pour les photos compressées, plutôt
 
 ### PNG (Portable Network Graphics)
 
-Le [PNG](/fr/docs/Glossary/PNG) (prononcé "**png**") utilise une compression sans perte ou avec perte afin de fournir une compression plus efficace, et prend en charge des profondeurs de couleurs plus élevées que [GIF](#gif), ainsi qu'une transparence alpha complète.
+Le format d'image [PNG](/fr/docs/Glossary/PNG) (prononcé "**png**") utilise une compression sans perte, prend en charge des profondeurs de couleurs plus élevées que le [GIF](#gif) et supporte la transparence alpha complète.
```

Comment:
```suggestion
Le format d'image [PNG](/fr/docs/Glossary/PNG) utilise une compression sans perte, prend en charge des profondeurs de couleurs plus élevées que le [GIF](#gif), est plus efficace, et prend complètement en charge la transparence alpha.
```

(tiens je découvre qu'en anglais on prononce "ping" 🤯)

Supporter est usité en français, mais est un faux ami pour le verbe anglais to support.

Reply:
addendum : j'ai retiré le "(prononcé "**png**")" parce qu'il ne me semble pas utile en français (au sens où c'est un acronyme composé uniquement de consonnes et où il n'y a pas vraiment d’ambiguïté...)